tests/unit_tests/config/test_queue_config.py

```diff
@@ -524,22 +528,34 @@ def test_wrong_generic_queue_option_raises_validation_error(queue_system, key, v
         f.write("NUM_REALIZATIONS 1\n")
         f.write(f"QUEUE_SYSTEM {queue_system}\n")
         f.write(f"QUEUE_OPTION GENERIC {key} {value}\n")
-    error_msg = (
-        "is not a valid positive integer."
-        if key == "MAX_RUNNING"
-        else "is not a valid integer or float."
-    )
-    with pytest.raises(ConfigValidationError, match=error_msg):
+    with pytest.raises(
+        ConfigValidationError, match="Input should be greater than or equal to 0"
+    ):
         ErtConfig.from_file("config.ert")
 
     with open("config.ert", mode="w", encoding="utf-8") as f:
         f.write("NUM_REALIZATIONS 1\n")
         f.write(f"QUEUE_SYSTEM {queue_system}\n")
         f.write(f"{key} {value}\n")
     error_msg = (
-        "must have a positive integer value as argument 1"
+        "must have a positive integer value"
         if key == "MAX_RUNNING"
-        else "is not a valid integer or float."
+        else "Input should be greater than or equal to 0"
     )
     with pytest.raises(ConfigValidationError, match=error_msg):
         ErtConfig.from_file("config.ert")
+
+
+@pytest.mark.parametrize(
+    "queue_system",
+    (QueueSystem.LSF, QueueSystem.TORQUE, QueueSystem.LOCAL, QueueSystem.SLURM),
+)
+def test_driver_initialization_from_defaults(queue_system):
```

Comment:
should there be an assert or something in the end?

Reply:
The intention of the test was just to ensure that we can initialize without errors. If there are any spelling errors in the parameter names to the drivers init() this test will fail, also if we provide too many parameters. Testing that the arguments are processed correctly inside the drivers should be the responsibility of the drivers, not the config tests.